envs/vpn/conda/fedbiomed-researcher.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
   - pip >= 23.0
   # TODO: consider migrating for classical notebooks to notebook7 or jupyterlab
   # https://jupyter-notebook.readthedocs.io/en/latest/migrate_to_notebook7.html
+  - notebook =6.*.*
```

Comment:
@ErwanDemairy it would be nice to keep tighter package specification as for other packages to minimize packager version change issue eg:

```
notebook ~=6.5.0
```

Pushing a proposed fix for that